Tests/Unit/Domain/Model/LanguageTest.php

```diff
@@ -2,8 +2,8 @@
 
 namespace Unit\Domain\Model;
```

Comment:
```suggestion
namespace WebVision\WvDeepltranslate\Tests\Unit\Domain\Model;
```
Should be indentical, I guess?